frontend/src/app/utils/dates/dates.helper.ts

```diff
@@ -0,0 +1,3 @@
+export const isBetween = (date: Date, start: Date, end: Date) => {
```

Comment:
Just for info, momentjs seems to have more functionality than date-fns, and such isBetween is included.
I didn't check library sizes, but for me momentjs is more comfortable.
https://momentjs.com/docs/